### DISCUSSION TITLE
인덱스명을 지을 때, 어떤 규칙을 따르면 좋을까?

### DISCUSSION THREAD

**Discussion by devjun10** [dailyge/dailyge-server#85] [Decision]
## 🔗 연관 PR

<br/><br/>

## ❓ 작성 이유

인덱스명 규칙은 **`팀마다 다르기 때문`** 에, 조금 더 좋은 규칙 을 찾기 위해 글을 작성하게 되었습니다. 

<br/><br/>

## 📝 내용

현재 프로젝트에서는 [해당 글](https://stackoverflow.com/questions/2783495/sql-server-index-naming-conventions)을 참조해서 다음과 같이 인덱스명을 사용하고 있습니다. 본인 의견이나 다른 방식이 있다면, 댓글로 남겨주세요. 

> idx_[TABLE_NAME]_[FIELDS...]

**Comment by beatmeJY** [comment]
저도 다른 몇 가지 글들을([1](https://www.quora.com/What-naming-convention-do-you-use-for-SQL-indexes), [2](https://stackoverflow.com/questions/2783495/sql-server-index-naming-conventions) ) 참조하였을 때, 가장 많이 나오는 의견이
- `idx_[TABLE_NAME]_[FIELDS...]`
- `ix_[TABLE_NAME]_[FIELDS...]`

로 많이 추려지는데 저도 `ix`보다는 `idx`가 더 인덱스에 가까운 느낌을 받는 것 같아 좋은 것 같습니다.👍🏻

**Reply by devjun10** [reply]
OK. 이대로 가시죠!
